pkgs/by-name/re/rescrobbled/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, bash
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, dbus
+}:
+
+rustPlatform.buildRustPackage rec {
+
+  pname = "rescrobbled";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "InputUsername";
+    repo = "rescrobbled";
+    rev = "v${version}";
+    hash = "sha256-1E+SeKjHCah+IFn2QLAyyv7jgEcZ1gtkh8iHgiVBuz4=";
+  };
+
+  cargoHash = "sha256-ZJbyYFvGTuXt1aqhGOATcDRrkTk7SorWXkN81sUoDdo=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl dbus ];
+
+  postPatch = ''
+    # Required for tests
+    substituteInPlace src/filter.rs --replace '#!/usr/bin/bash' '#!${bash}/bin/bash'
+  '';
+
+  postInstall = ''
+    substituteInPlace rescrobbled.service --replace '%h/.cargo/bin/rescrobbled' "$out/bin/rescrobbled"
+    install -Dm644 rescrobbled.service -t "$out/share/systemd/user"
+  '';
+
+  meta = with lib; {
+    description = "MPRIS music scrobbler daemon";
+    homepage = "https://github.com/InputUsername/rescrobbled";
+    license = licenses.gpl3Only;
```

Comment:
https://github.com/InputUsername/rescrobbled/blob/d9837ad4ddbe4f77a06c4f8a697d7d6df858e414/src/main.rs#L7-L8

Reply:
I'll change that now. Should I also note you down as a contributor on this package?

Reply:
I only packaged it to test it but did not end up using it.